app/src/models/popup.ts

```diff
@@ -65,7 +65,6 @@ export enum PopupType {
   CreateTag,
   DeleteTag,
   LocalChangesOverwritten,
-  RebaseConflicts,
```

Comment:
This is unrelated to this PR but I stumbled upon it trying to find the right dialog and it confused the heck out of me so I figured we should just remove it